examples/hashsmt/sha1.hpp

```diff
@@ -30,9 +30,8 @@
 // Note: this implementation does not handle message longer than
 //       2^32 bytes.
 
-#pragma once 
+#pragma once
```

Comment:
Since we are not using `#pragma once`, maybe change it to an `#ifdef` guard instead of removing the whitespace?